src/tests/receiving/assertQtyInputs.test.ts

```diff
@@ -0,0 +1,276 @@
+import { ShipmentType } from '@/constants/ShipmentType';
+import { expect, test } from '@/fixtures/fixtures';
+import { StockMovementResponse } from '@/types';
+import { formatDate, getDateByOffset } from '@/utils/DateUtils';
+
+test.describe('Assert qty inputs when split lines', () => {
```

Comment:
What are you "asserting" here?

Reply:
Plus maybe it would be better to use full `quantity` instead `qty` in these descriptions